apps/website/next.config.ts

```diff
@@ -5,5 +5,6 @@ const nextConfig = {
   serverRuntimeConfig: {
     maxDuration: 300,
   },
+  transpilePackages: ["@repo/database", "@repo/utils", "@repo/ui"],
```

Comment:
What problem is this solving related to "Move some functions to packages/database"?